### PR TITLE
Documentation for ROCR on Read the Docs

### PR DESCRIPTION
**Pull request by samjwu**
Use rocm-docs-core template for documentation

Rationale:
 - https://github.com/RadeonOpenCompute/rocm-docs-core#purpose

### CODE REVIEW

File: include/Brig.h [hunk 1]
```diff
@@ -0,0 +1,1131 @@
+// University of Illinois/NCSA
```
Comment (dayatsin-amd) [inline include/Brig.h:1]: I am ok with moving this file and other files from src/inc/ to include.

But overall, this patch series is not removing the files from the other directories. Same applies to the other files below.

**Review by dayatsin-amd — COMMENTED** [review]
Hi @AlexVlx and @samjwu 
Sorry for taking long time to review. It seems that your editor is using Windows line terminations. Can you please change to Linux line terminations.

**Review by dayatsin-amd — COMMENTED** [review]
Can you squash this commit with the previous commit that added these 2 files

**Review by dayatsin-amd — COMMENTED** [review]
Can you squash this with the commit: "add sphinx configs"

**Review by dayatsin-amd — COMMENTED** [review]
Please squash with the commits that accidentally introduced the incorrect line terminations

**Comment by samjwu** [comment]
Commits have been squashed

**Comment by dayatsin-amd** [comment]
There is still something wrong with the commits. This PR should have only 2 commits:

1. The commit that changes the file locations to align with Pitchfork layout.
    This commit should only have the changes for the file locations. It should not have changes to the contents of existing header files, e.g the change here: https://github.com/RadeonOpenCompute/ROCR-Runtime/pull/156/commits/7378d73a6b6193bc0cedb4e263de663b6cdd2ea9#diff-f6f85d43702fc574ac39e02835ad0b91ccd7c5b534ced40b69978fdce027f587
should not be there.

2. The commit that adds sphinx configuration files and only fix the formatting in files such as README.md. Similarly, it should not have changes to the contents of the existing header files.

**Comment by saadrahim** [comment]
> Commits have been squashed

Hi @samjwu,
Please amend the squash comment to say what the change is. No reference to squash and non existing comments are needed.

**Comment by samjwu** [comment]
Fixed in latest.

Extra changes to header files have been removed.

There are 2 commits:
1. The original by Alex
2. Configure sphinx

I cannot squash commit that merges with master in interactive rebase mode

**Comment by samjwu** [comment]
Merged in Gerrit

**Pull request closed without merging**